cmd/plural/deploy.go

```diff
@@ -428,6 +436,21 @@ func (p *Plural) destroy(c *cli.Context) error {
 		return git.Sync(repoRoot, commit, c.Bool("force"))
 	}
 
+	if confirm("Would you like to remove installations from app.plural.sh?") {
+		if err := p.handleResetInstallations(c); err != nil {
+			return err
+		}
+		man, err := manifest.FetchProject()
+		if err != nil {
+			return err
+		}
+		if err := p.DestroyCluster(man.Network.Subdomain, man.Cluster, strings.ToUpper(man.Provider)); err != nil {
```

Comment:
We can actually just always call this api w/o confirmation, it should be safe.

That said, we should only do it when it's not a single app destroy (so check `repoName != ""`)